lib/std/Build/Step/Run.zig

```diff
@@ -1233,16 +1252,8 @@ fn evalGeneric(self: *Run, child: *std.process.Child) !StdIoResult {
             child.stdin = null;
         },
         .lazy_path => |lazy_path| {
-            const path = lazy_path.getPath(self.step.owner);
-            const file = self.step.owner.build_root.handle.openFile(path, .{}) catch |err| {
-                return self.step.fail("unable to open stdin file: {s}", .{@errorName(err)});
-            };
-            defer file.close();
-            child.stdin.?.writeFileAll(file, .{}) catch |err| {
-                return self.step.fail("unable to write file to stdin: {s}", .{@errorName(err)});
-            };
-            child.stdin.?.close();
-            child.stdin = null;
+            _ = lazy_path;
+            // This was moved to earlier since it's done on start, will test
         },
         .none => {},
```

Comment:
`.lazy_path, .none => {}`

Reply:
plus if it isn't tested, maybe make sure it works as intended?

Reply:
:+1:

Reply:
Oh, and since I didn't clarify, I did run the behaviour and stdlib test suite